lib/rules/space-infix-ops.js

```diff
@@ -8,6 +8,23 @@
 // Rule Definition
 //------------------------------------------------------------------------------
 
+const OPERATORS = [
```

Comment:
Wouldn't we have to find a way to get that list from all parsers since we can't assume Espree is being used?